cli/install/install.go

```diff
@@ -448,13 +448,13 @@ func installTt(version string, binDir string, flags InstallationFlag, distfiles
 			if err != nil {
 				return err
 			}
-			util.ExecuteCommand("git", flags.Verbose, logFile, path, "checkout", "master")
+			util.ExecuteCommand("git", flags.Verbose, logFile, path, "checkout", ttVersion)
```

Comment:
Need to cover this with test to avoid such errors in future.

Reply:
Done